src/components/pages/improve-communication-experience/painRestatement/painRestatement.jsx

```diff
@@ -6,28 +6,29 @@ import missed from './images/missed.svg';
 import negative from './images/negative.svg';
 import notifications from './images/notifications.svg';
 
-const TITLE = 'Effective Notification Infrastructure';
-const SUBTITLE = 'Users can tell when a notification system is poorly designed or managed.';
+const TITLE = 'Poor notifications experiences hurt your business';
+const SUBTITLE =
+  'End users expect seamless, on-brand, and timely notifications that add to their experience.';
 
 const CARDS = [
   {
-    title: 'Missed Updates',
+    title: 'Missed updates',
     description:
-      'Without timely notifications, users may miss critical updates or events related to their interests or activities, which may lead to frustration.',
+      'Without timely notifications, users miss critical information or updates such as order updates, password resets, or one-time password messages, leading to a poor experience with your application.',
     image: <img width={80} height={80} src={missed} alt="Create template" loading="lazy" />,
   },
   {
-    title: 'Irrelevant Notifications',
+    title: 'Notification fatige',
     description:
-      'Poorly targeted or excessive notifications can overwhelm users, causing annoyance and distraction from their intended tasks or activities.',
+      'Poorly targeted or excessive notifications overwhelm and annoy users, causing distractions and encouraging them to ignore them altogether.',
```

Comment:
maybe instead of

causing distractions and encouraging them to ignore them altogether

we use

causing distractions and encouraging them to ignore notifications altogether

(the double them was a little unclear)

Reply:
Yep, another good suggestion.